sda/processor_test.go

```diff
@@ -173,6 +183,14 @@ func procMsg(si *network.ServerIdentity, msg *testMsg) (network.Body, error) {
 	return msg, nil
 }
 
+func procMsg2(si *network.ServerIdentity, msg *testMsg) (network.Body, error) {
+	// Return an error for testing
+	if msg.I != 42 {
+		return nil, errors.New("6 * 9 != 42")
```

Comment:
There is already this "6 * 9 != 42" somewhere else I think... I know you're a fan of 42 but **if it's a duplicate** can you change so we can find easily where does the error comes from ?